samples/scripts/build-logs

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+if [ $# -eq 1 ]; then
+    BUILD_RUN_NAME=$1
+    FOLLOW=0
+elif [ $# -eq 2 ] && [ "$2" = "--follow" ]; then
+    BUILD_RUN_NAME=$1
+    FOLLOW=1
+else
+    echo "Usage: build-logs <BUILD_RUN_NAME> [--follow]"
+    exit 1
+fi
+
+BUILD_RUN_NAME=$1
+
+# Retrieve the buildrun
+
+BUILD_RUN=`kubectl get buildrun ${BUILD_RUN_NAME} -o json 2>/dev/null`
+
+if [ $? -ne 0 ]; then
```

Comment:
You could run the command directly on the `if` without adding it to a new variable, e.g.
```sh
if ! kubectl get -n build-examples buildrun "${BUILD_RUN_NAME}" -o json >/dev/null 2>&1 ; then
fi
```

Reply:
Done.